lib/main.dart

```diff
@@ -15,15 +18,50 @@ class MyApp extends StatelessWidget {
         fontFamily: "Raleway",
         primarySwatch: Colors.teal
       ),
-      home: HomePage(),
+      home: Router(),
```

Comment:
Have you been learning anything new? I feel like your design patterns change a lot. Trying to figure out if it is due to the fact that you are learning or hacking.

Reply:
Yes I changed the design here to mimic fragments in android. I moved the Home Page so that it will act as the main activity for the fragments.